quickstarts/microsoft.netapp/anf-sap-hana/anf-sap-hana-storage/azuredeploy.json

```diff
@@ -0,0 +1,618 @@
+                "proximityPlacementGroup": "[resourceId(parameters('ProximityPlacementGroupResourceGroup'),'/providers/Microsoft.Compute/proximityPlacementGroups', parameters('ProximityPlacementGroupName'))]",
+                "usageThreshold": "[mul(variables('1GiBInBytes'),variables('logBackupVolumeSize'))]",
+                "capacityPoolResourceId": "[resourceId('Microsoft.NetApp/netAppAccounts/capacityPools', parameters('ANFNetappAccount'), parameters('CapacityPool'))]",
+                "kerberosEnabled": false,
+                "volumeSpecName": "log-backup",
+                "throughputMibps": "[variables('logBackupVolumeThroughput')]",
+                "subnetId": "[resourceId('Microsoft.Network/virtualNetworks/subnets', parameters('VirtualNetwork'), parameters('DelegatedSubnet'))]",
+                "exportPolicy": {
+                    "rules": [
+                        {
+                            "ruleIndex": 1,
+                            "unixReadOnly": false,
+                            "unixReadWrite": true,
+                            "cifs": false,
+                            "nfsv3":  "[variables('isNFSv3Enabled')]",
+                            "nfsv41": "[variables('isNFSv41Enabled')]",
+                            "allowedClients": "0.0.0.0/0",
+                            "kerberos5ReadOnly": false,
+                            "kerberos5ReadWrite": false,
+                            "kerberos5iReadOnly": false,
+                            "kerberos5iReadWrite": false,
+                            "kerberos5pReadOnly": false,
+                            "kerberos5pReadWrite": false,
+                            "hasRootAccess": true
+                        }
+                    ]
+                },
+                "protocolTypes": [
+                    "[parameters('DataBackupAndLogBackupNFSVersion')]"
+                ]
+            }
+        },
+
+        "emptyArray": "[createArray()]",
+        "dataLogSharedPayloadArray": "[concat(array(variables('dataVolumePayload')), array(variables('logVolumePayload')),if(equals(parameters('SharedSizeInGibibytes'), 'none'), variables('emptyArray'), array(variables('sharedVolumePayload'))))]",
+        "dataLogSharedDataBackupPayloadArray": "[concat(variables('dataLogSharedPayloadArray'), if(equals(parameters('DataBackupSizeInGibibytes'), 'none'), variables('emptyArray'), array(variables('dataBackupVolumePayload'))))]",
+        "masterHostVolumesPayloadArray": "[concat(variables('dataLogSharedDataBackupPayloadArray'), if(equals(parameters('LogBackupSizeInGibibytes'), 'none'), variables('emptyArray'), array(variables('logBackupVolumePayload'))))]"
+    },
+    "resources": [
+        {
+            "type": "Microsoft.NetApp/netAppAccounts/volumeGroups",
+            "apiVersion": "2021-06-01",
+            "comments": "Volume Group creation for master Host. This will be deployed only if Starting SAP HostID ==1 i.e master Host",
```

Comment:
See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties for the proper sort order of properties

Reply:
done